lobby/src/main/java/org/triplea/lobby/server/login/forgot/password/create/PasswordEmailSender.java

```diff
@@ -0,0 +1,57 @@
+package org.triplea.lobby.server.login.forgot.password.create;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Properties;
+import java.util.function.BiConsumer;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import org.triplea.lobby.server.EnvironmentVariable;
+
+/** Sends a temporary password to a target user. */
+class PasswordEmailSender implements BiConsumer<String, String> {
+  private static final String FROM = "no-reply@triplea-game.org";
+
+  @Override
+  public void accept(final String email, final String generatedPassword) {
+    if (EnvironmentVariable.LOCAL_DEV.getBoolean()) {
+      // do not send emails from local dev.
```

Comment:
I had good experiences with https://ethereal.email/ which is a no-op SMTP server, you can use to verify everything works.
You might want to be allow configuring such a service here

Reply:
That's a pretty neat service. If it were simply a matter of swapping out the configured host, that would be appealing. On the other hand if we have to add any additional code to work with it, then we're no longer really testing the original mail sending code and have introduced a new liability and point of complexity that is not actually verifying the prod code. It's also a benefit to avoid network calls in local dev.

The flag is kinda hacky, but it is straight forward and effective. If we can overcome the problems I stated above, and configure this cleanly (and we'd still need a flag to turn off the network communication), I wouldn't necessarily have objection. IMO it's better to focus on delivering the feature for now and revisiting later as we have time and desire.